webapp/sources/rudder/rudder-web/src/main/resources/configuration.properties.sample

```diff
@@ -224,6 +224,22 @@ rudder.batch.purge.inventories.delete.TTL=7
 # Interval in hours
 rudder.batch.purge.inventories.delete.interval=24
 
+###########################
+# Promise folder cleaning ###########################################################
+###########################
+
+#
+# Promise folder cleaning
+# This allows you to schedule the purge of unused promised folders (Deleted Nodes, or Nodes that has moved to another relay/policy server)
```

Comment:
This allows you to schedule s/the// purge of unused s/promised/policies folders (s/D/d/eleted s/N/n/odes, 
or s/N/n/odes that s/has// moved to another relay or policy server)